tentacoli.js

```diff
@@ -291,6 +294,21 @@ Tentacoli.prototype.request = function (data, callback) {
   return this
 }
 
+Tentacoli.prototype.fire = function (data, callback) {
+  var that = this
+  var req = new Request(null, callback)
+
+  try {
+    wrapStreams(that, data, req, true)
+  } catch (err) {
+    return this
```

Comment:
There are two options (both valid):

a. remove the `try/catch`, and blow up the exception to the user
b. add a comment here why we are swallowing the exception

The safest option might be a, but b. is also ok.